packages/fluentui/react-northstar/src/utils/positioner/Popper.tsx

```diff
@@ -292,6 +294,19 @@ export const Popper: React.FunctionComponent<PopperProps> = props => {
           fn: handleUpdate,
         },
         popperInitialPositionFix.modifier,
+
+        autoSize && maxSize,
+        autoSize && {
+          name: 'applyMaxSize',
+          enabled: true,
+          phase: 'beforeWrite' as PopperJs.ModifierPhases,
+          requires: ['maxSize'],
+          fn({ state }) {
+            const { height, width } = state.modifiersData.maxSize;
+            state.styles.popper.maxHeight = `${height}px`;
+            state.styles.popper.maxWidth = `${width}px`;
```

Comment:
Is it enough for other components that have this in their interface? For example, `ToolbarItem`, `Dropdown`, etc.